package.json

```diff
@@ -44,8 +44,8 @@
     ]
   },
   "scripts": {
-    "start": "bash -c \"./run_server.sh\"",
-    "stop": "bash -c \"./stop_server.sh\"",
+    "start": "bash ./run_server.sh",
+    "stop": "bash ./stop_server.sh",
```

Comment:
Does the `./` still work on windows? Would `bash run_server.sh` work better if not?